plugins/local_application/workspace.py

```
@@ -0,0 +1,633 @@
+            binding = _binding(project_id, config_digest, profile_digest, initialized_at=clock.now())
+            app.close()
+            app = None
+            _atomic_json_write(internal / BINDING_NAME, binding)
+            marker.unlink()
+            return cls.open(root)
+        except Exception:
+            if app is not None:
+                try:
+                    app.close()
+                except Exception:
+                    pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)